src/jit/function-builder.cc

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2017 wasmjit-omr project participants
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "function-builder.h"
+#include "src/interp.h"
+#include "ilgen/VirtualMachineState.hpp"
+#include <type_traits>
+
+wabt::jit::FunctionBuilder::FunctionBuilder(interp::Thread* thread, interp::IstreamOffset const offset, TypeDictionary* types)
+    : TR::MethodBuilder(types),
+      thread_{thread},
+      offset_{offset},
+      valueType_{types->LookupUnion("Value")},
+      pValueType_{types->PointerTo(types->LookupUnion("Value"))}
```

Comment:
WABT seems to use exclusively parentheses for initializing objects, so we should probably stick to that convention where possible.